.changeset/hot-jars-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/next-on-pages": patch
+---
+
+Overwrite prerendered files if they already exist in the output directory
```

Comment:
Could you mention that this is so that `--skip-build` can cope with prerendered files? 🙏